### DISCUSSION TITLE
Use the same item delegate for multiple lists

### DISCUSSION THREAD

**Discussion by khaitranhq** [charmbracelet/bubbles#628] [Q&A]
Hi everyone, 
I'm new to this amazing framework and currently working on a tool that involves multiple lists. I noticed that I can't use the same item delegate across different lists. The first list works fine, but the others don't display any items. I'm not sure if I'm overlooking something or if this behavior is intentional. Any insights would be appreciated!

**Comment by activey** [comment]
Code?
